### ISSUE TITLE
Google Access code link broken

### ISSUE THREAD

**Issue by ondemandtasks** [awesomemotive/Google-Analytics-Dashboard-for-WP#156]
When attempting to setup plugin with my Google info.

I was able to setup everything except the access code.

When I click on the access code link I get an invalid link error from google

I have client ID and Client completed then when I click on authorize plugin it prompts me to add one time access code.

When I click on get access code I get an error 400 Error: redirect_uri_mismatch.

Here is the url that the "Get access code" is opening
https://accounts.google.com/o/oauth2/auth?response_type=code&redirect_uri=urn%3Aietf%3Awg%3Aoauth%3A2.0%3Aoob&client_id=794343755409-s27n7daj4tf1nr517d9ldi8luah7n0rn.apps.googleusercontent.com&scope=https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fanalytics.readonly&access_type=offline&approval_prompt=auto

So basically I am needing direction on obtaining my access code so I can complete the setup.

![image](https://user-images.githubusercontent.com/59511813/71769482-a3e15c80-2ee7-11ea-8f07-e85110aa7f63.png)
